homeassistant/components/screenlogic/sensor.py

```diff
@@ -231,15 +256,15 @@ def native_value(self):
         return (value - 1) if "supply" in self._data_key else value
 
     @property
-    def sensor(self):
+    def sensor(self) -> dict:
```

Comment:
Can this be dict[str, Any]?

Reply:
Possibly. There are some places where the key is an int, but maybe not where these would reference. I'd have to check.

Reply:
Could also be `dict[str | int, Any]`